tests/integration_tests/test_fulltext_parser_config.py

```
@@ -153,7 +154,7 @@ def _test_backward_compatibility(self, client):
         assert collection is not None
 
         # Verify default parser (ik) is used for backward compatibility
-        table_name = f"c$v1${test_collection_name}"
+        table_name = CollectionNames.table_name(test_collection_name)
```

Ruff (0.14.10)

Undefined name `CollectionNames`

(F821)